packages/core/src/docs/colors.md

```diff
@@ -73,4 +73,3 @@ scheme that does not imply order, merely difference in kind.
 
 @reactDocs QualitativeSchemePalette
 
-@include color-aliases
```

Comment:
I'm not sure I follow, is it because you want the old link to redirect to the new one? if so, that'd be neat, but ideally we don't have to duplicate sections in different places, it just redirects

Reply:
@llorca i pushed a commit, check out the updated docs